### ISSUE TITLE
restructure blas/math out of tensors

### ISSUE THREAD

**Issue by grahamlopez** [project-asgard/asgard#72]
ASGarD
    asgardasgard
    Issues
    #72

Open
Opened 3 weeks ago by Graham Lopez
Close issue
New issue
restructure blas/math out of tensors

we want to start using some blas/lapack as free functions, but we want nice abstraction points over their providers.

    we want to have a single entry point to whatever flavor of vendor lib we use
    we want to use our types instead of extracting pointers to pass in

so we propose to create a new lower-level component ("wrapped_blas", "math_utils", whatever) as well as a new "free_blas" or similarly-named component. Both tensors and free_blas will depend on wrapped_blas.
Related issues
0
Graham Lopez @bgl added hygeine label 3 weeks ago
Graham Lopez
Graham Lopez @bgl · 2 weeks ago
Owner

(waiting on timeloop merge to start this fixup)

**Comment by grahamlopez** [comment]
closed by #95
